spec/support/api_analytics_shared_examples.rb

```diff
@@ -23,7 +23,7 @@
   end
 
   context "when a valid token is given but the user is not a pro" do
-    before { get "/api/analytics/#{endpoint}?#{params}", headers: { "api-key" => "abadskajdlsak" } }
+    before { get "/api/analytics/#{endpoint}?#{params}", headers: { "api-key" => api_token.secret } }
```

Comment:
we weren't actually testing what we said we were testing, thus this was identical with another case